assets/humans.txt

```diff
@@ -1,103 +1,107 @@
-wellcomecollection.org has been made by:
+Wellcomecollection.org has been made byÖ
+
 _______________________________________________________________________________
 
 
 /* WE ARE */
 
-Digital Engagement Office:
-	Tom Scott
-	Hannah Brown
-
-Digital Production (digitisation):
-	Christy Henshaw
-	Harkiran Dhindsa
-	Branwen Lloyd-Evans
-	Holly Peel
-	Rosie Curran
-	Paul Horn
-	Cassidy Phillips
-	Emilie Bell
-	Daila Malafronte
-	Ashley Ray
-
-Digital Editorial Content:
-	Danny Birchall
-	Kirsten Riley	
-	Lalita Kaplish
-	Alice White
-	Kate Wilkinson 
-	Helen Babbs
-	Kate Quarry
-	James Machin
-	Richard Sheehan
-	Aisha Mazhar
-	Robert Bidder
-
-Photography:	
-	Richard Everett	
-	Ben Gilbert
-	Thomas S G Farnetti
-	Laurie Auchterlonie
-	Steven Pocock
-	Thomas Cox	
-	Kathleen Arundell	
-
-Digital Experience:
-	Jennifer Phillips-Bacher
-	Tacey Laurie
-	Gareth Ormerod 
-	Lyndsay Thackrah
-	Gareth Estchild
-	David McCormick
-	Dominique Marshall
-	Alice Richmond
-
-Digital Preservation and Data:
-	Jonathan Tweed
-	Natalie Pollecutt
-	James Gorrie
-	Alice Fuzier-Cayla
-	Alex Chan
-	Robert Kenny
-	Harrison Pim
-	Jamie Parkinson	
-	Nick Ward
-	Helen Sherwood-Taylor
-
-Access:
-	Selene Burn & Orla O'Donnell
+	Department Head:		Tom Scott
+	Operations Officer:		Hannah Brown
+
+Digital Production (digitisation)
+	Digital Production Manager:	Christy Henshaw
+	A/V Digitisation Manager:	Harkiran Dhindsa
+	Metadata Analyst:		Branwen Lloyd-Evans
+	Digitisation Services Manager:	Holly Peel
+	Digitisation Team Leader:	Rosie Curran
+	Digitisation Support Officer:	Paul Horn
+	Digitisation Support Officer:	Cassidy Phillips
+	Digitisation Support Officer:	Emilie Bell
+	Digitisation Support Officer:	Daila Malafronte
+	Digital Workflow Specialist:	Ashley Ray
+
+Digital Editorial Content
+	Digital Content Manager:	Danny Birchall
+	Digital Editor SOCIAL MEDIA:	Kirsten Riley
+	Digital Editor:			Lalita Kaplish
+	Digital Editor & Wikimedian in Residence: Alice White
+	Digital Editor:			Kate Wilkinson
+	Digital Editor:			Helen Babbs
+	Digital Editor:			Anjuli Sharma
+	Copy Editor & Proofreader:	Kate Quarry
+	Copy Editor & Proofreader:	James Machin
+	Copy Editor & Proofreader:	Richard Sheehan
+
+
+Photography
+	Photographic Manager: 		Richard Everett
+	Senior Photographer & Picture Editor: Ben Gilbert
+	Photographer:			Thomas S G Farnetti
+	Senior Photographer:		Laurie Auchterlonie
+	Photographer:			Steven Pocock
+	Photographer:			Thomas Cox
+	Project Photographer:		Kathleen Arundell
+
+Digital Experience
+	Digital Analyst:	        Tacey Laurie
+	Senior User Interface Designer:	Gareth Ormerod
+	Product Designer:		Dominique Marshall
+	User Researcher:		Alice Richmond
+	Senior Software Developer:	Gareth Estchild
+	Software Developer:		David McCormick
+
+Digital Preservation and Data
+	Technical Product Manager:	Jonathan Tweed
+	Technical Delivery Manager:	Natalie Pollecutt
+	Name: 				James Gorrie
```

Comment:
@jamesgorrie I'm tidying up the spacing in this file and spotted this. Are you a 'Senior Software Developer'?

Reply:
I'd be happy as developer.

Reply:
your wish is my command

Reply:
…actually, latest update means nobody gets a job title which I guess (hope) you'll also be happy with.